src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1388,7 +1388,8 @@ void ApplicationManagerImpl::SendMessageToMobile(
   if (msg_to_mobile[strings::params].keyExists(strings::correlation_id)) {
     request_ctrl_.OnMobileResponse(
```

Comment:
@AByzhynar  see 9d172a2. Please try to post all review suggestions in one iteration.